build-wheelhouse/action.yml

```
@@ -182,9 +190,10 @@ runs:
         retention-days: 7
 
     - name: Check library's dependencies license
-      uses: ansys/actions/check-licenses@main
+      uses: ansys/actions/check-licenses@feat/whitelist-packages
```

Should be reverted to main once approved and before merging
```suggestion
      uses: ansys/actions/check-licenses@main
```